src/js/node/tls.ts

```diff
@@ -607,8 +622,11 @@ const DEFAULT_ECDH_CURVE = "auto",
   DEFAULT_MIN_VERSION = "TLSv1.2",
   DEFAULT_MAX_VERSION = "TLSv1.3";
 
+const DEFAULT_CIPHERS_LIST = DEFAULT_CIPHERS.split(":");
+const DEFAULT_CIPHERS_SET = new Set([...DEFAULT_CIPHERS_LIST.map(c => c.toLowerCase()), ...DEFAULT_CIPHERS_LIST]);
+
 function normalizeConnectArgs(listArgs) {
-  const args = net._normalizeArgs(listArgs);
+  const args = (net as any)._normalizeArgs(listArgs);
```

Comment:
would be nice if we added this to our type overrides

Reply:
Fixed in eed875c7090ef1cfa2c953a309f153192bece203